modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/IgniteH2Indexing.java

```diff
@@ -1748,6 +1760,112 @@ public UpdateResult executeUpdateOnDataNode(
         return schemaMgr.schemaNames();
     }
 
+    /** {@inheritDoc} */
+    @Override public Collection<TableInformation> tablesInformation(String schemaNamePtrn, String tblNamePtrn,
+        String... tblTypes) {
+        Set<String> types = F.isEmpty(tblTypes) ? Collections.emptySet() : new HashSet<>(Arrays.asList(tblTypes));
+
+        Collection<TableInformation> infos = new ArrayList<>();
+
+        if (F.isEmpty(tblTypes) || types.contains(TableType.TABLE.name())) {
```

Comment:
I suggest to introduce `boolean allTypes` variable instead of using `F.isEmpty(tblTypes)` explicitly.

Reply:
ok